app/src/main/java/org/wikipedia/usercontrib/UserContribItemView.kt

```diff
@@ -0,0 +1,63 @@
+package org.wikipedia.usercontrib
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.isVisible
+import org.wikipedia.R
+import org.wikipedia.databinding.ItemUserContribBinding
+import org.wikipedia.dataclient.mwapi.UserContribution
+import org.wikipedia.util.DateUtil
+import org.wikipedia.util.ResourceUtil
+import org.wikipedia.util.StringUtil
+
+class UserContribItemView(context: Context) : FrameLayout(context) {
+    interface Listener {
+        fun onClick()
+        fun onLongClick()
```

Comment:
Do we need to implement the `onLongClick` action for the list item?

Reply:
Nope! Removed.